test/TestAsset/zendview-custom-suffix.pht

```diff
@@ -0,0 +1,3 @@
+<h1>This is a template file for ZendView with <em>php</em> suffix</h1>
```

Comment:
Should this read "php" or "pht"? the file itself has a "pht" suffix...

Reply:
yes, right...sorry... it was a ".php" extension originally...